actions/recipe_action.go

```diff
@@ -60,15 +61,19 @@ func (recipe *RecipeAction) Verify(context *debos.DebosContext) error {
 
 	// Add Variables to template vars
 	for k, v := range recipe.Variables {
+		if k == "architecture" {
```

Comment:
This is weird. Also the commit message doesn't explain much of what this is intending to achieve.